hedyorg/hedy
# [TESTING] 3830 testing create front end tests for the admin page

**Description**

This includes the front-end tests for the admin overview page (`/admin`)

**Fixes _issue or discussion number_**

Fixes #3830 

**How to test**

Run 
`export ADMIN_USER=admin; python app.py` in your virtual environment. This ensures that there is an active admin and Hedy is running.

Then you can go to the /tests folder and run: 

`npx cypress run --spec cypress/e2e/admin_page/admin_overview_page`

The specs should have passed!
